### DISCUSSION TITLE
(unknown) : Type not found : openfl.utils._internal.ExtraParamsMacro

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#440] [Q&A] [answered]
Ran update.bat and then got this error. Solution?
![image](https://github.com/user-attachments/assets/89d1e959-352e-4505-ab48-990694389dfb)

**Comment by THEkatinamicrowave** [comment]
@NeeEoo pleeease don't ghost me for a week after being so helpful 😭

**Comment by SrtHero278** [comment]
oh yeah that just happens

**Reply by THEkatinamicrowave** [reply]
ok... how do I fix it?

**Comment by NexIsDumb** [comment, marked as answer]
thats not something about us, but about the openFL lib which is not ours
you can try to deleting the .haxelib folder and run the bat again

**Reply by THEkatinamicrowave** [reply]
Veritatem mihi ostendisti
Ago multas gratias

**Reply by NexIsDumb** [reply]
oh wow didnt expect it to work so fast😭

**Reply by THEkatinamicrowave** [reply]
dw if this was a movie we'd have to freaking break into HaxeFlixel Studios or some sht n steal the code

**Reply by NexIsDumb** [reply]
😭
